.github/workflows/build-test.yaml

```diff
@@ -25,6 +25,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```

Comment:
Can you make commit message for this change more descriptive? What consequences this change has, does it impact build time?

Reply:
Right, what this line does is, it fetches all history for all branches and tags. The default `fetch-depth: 1` only fetches the first commit. So in order to get the latest tag, I need a more extensive history. I could limit it to few 100s, but I believe that this change shouldn't take more than a 1s.